CHANGELOG.md

```diff
@@ -345,6 +345,8 @@ v1.6.0
 
 - Fixed issue with automemlimit logging bad messages and trying to access cgroup on non-linux builds (@dehaansa)
 
+- Fixed an issue in the `mimir.rules.kubernetes` component that would keep the component as unhealthy even when it managed to start after temporary errors (@nicolasvan)
```

Comment:
This line would need to go to the "main branch" section of the changelog.

Reply:
Of course! Fixed it.